### DISCUSSION TITLE
How to update ckeditor from 4.22.1 version to 4.24.0-lts in pimcore 10.6 version

### DISCUSSION THREAD

**Discussion by Thejaswini-Rao-U** [pimcore/pimcore#17327] [Admin-Ui-Classic-Bundle]
### Pimcore version

10.6.9

### Steps to reproduce

Hi
In my project, sometimes in the object i see below message appearing 

" **This CKEditor 4.22.1 version is not secure. Consider "https://ckeditor.com/ckeditor-4-support/" 4.24.0-lts.**"

how to update ckeditor version from 4.22.1 to 4.24.0 ? i tried to find out in pimcore document or anywhere i did not find updating command or procedure for this?

please let me know how to solve this?

Thanks in advance
Thejaswini

### Actual Behavior

gives warning or error in admin interface object

### Expected Behavior

how to update

**Comment by blankse** [comment]
@Thejaswini-Rao-U You can deactivate the version check. See https://github.com/pimcore/pimcore/issues/16574

**Reply by Thejaswini-Rao-U** [reply]
@blankse Its working for me

Thanks

**Reply by AlephLau** [reply]
@blankse "But is the best solution just to delete this message?
Does this version have no security vulnerabilities, or why did they decide to show the message to update to LTS?

**Reply by blankse** [reply]
@AlephCode Pimcore 10.6 also have security vulnerabilities. So when you want a safe system you have to update to Pimcore 11 with Tinymce. I think they decide to show the message to sell the LTS version ;)

Otherwise you have to pay Pimcore 10.6 LTS and Ckeditor 4.24 LTS

**Reply by AlephLau** [reply]
@blankse Don't worry, I also think they show that message just to sell the LTS haha

**Comment by AlephLau** [comment]
In Pimcore version 11, did they take an alternative to this dependency, considering that CKEditor 4.22.1 is the latest free version?

**Reply by natted** [reply]
I think Pimcore 11 switched to tinymce
